apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/RequestResponsePopulateMdcFilter.java

```diff
@@ -174,12 +176,18 @@ else if (headerValues.size() == 1)
       }
     }
 
+    // set to null
+    request.setAttribute(RESPONSE_DURATION_MILLISECONDS, null);
```

Comment:
don't think we need this since the caller is checking for null on the attribute

Reply:
removed